index.markdown

```diff
@@ -1,12 +1,66 @@
 ---
-title: Savile - Turing's Design System
+title: Savile
 layout: default
 ---
 
-Welcome to Savile.
+## Turing School's Design System
 
-{% for example in site.example %}
+Nicknamed after <a class="s-link" href="https://en.wikipedia.org/wiki/Savile_Row" target="blank">Savile Row</a>, Savile is the Design System for all sites and applications in the Turing Software suite. It is open-sourced though, so available for any who are interested to use!
 
-  <h2>{{ example.name }} - {{ example.position }}</h2>
-  <p>{{ example.content | markdownify }}</p>
-{% endfor %}
+<br>
+
+### Adding Savile to an Application
+
+Savile is language and framework agnostic, so no matter what an application is built in, it can utilize this design system. This means that the setup details may vary based on the framework.
+
+To add Savile to a project, import the URL of the version you'd like to use in the appropriate place. 
+
+<table class="spaced-table">
+  <thead>
+    <tr class="table-head-row">
+      <th>Version Number</th>
+      <th>URL</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr class="table-row">
+      <td class="token-title-cell">1.1 - <em>latest stable release</em></td>
+      <td class="token-title-cell">
+        <a class="s-link" href="https://savile.turing.io/css/v1/1.1.css" target="blank">https://savile.turing.io/css/v1/1.1.css</a>
+      </td>
+    </tr>
+    <tr class="table-row">
+      <td class="token-title-cell">Latest - <em>only recommended for experimental use</em></td>
+      <td class="token-title-cell">
+        <a class="s-link" href="https://savile.turing.io/css/1-latest.css" target="blank">https://savile.turing.io/css/1-latest.css</a>
+      </td>
+    </tr>
+    <tr class="table-row">
+      <td class="token-title-cell">1.0</td>
+      <td class="token-title-cell">
+        <a class="s-link" href="https://savile.turing.io/css/v1/1.0.css" target="blank">https://savile.turing.io/css/v1/1.0.css</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+Simply importing Savile should not make any changes to your existing application. _When Savile is added to an existing Turing application, you may notice very small changes in the weight of some fonts. This is because some of the older applications use older versions of font files, and Savile will override the font source._
+
+<br>
+
+### Using Savile in an Application
+
+Once Savile is added to your project, variables, class names, and assets will be available anywhere in your project. 
+
+In the documentation, you will find:
+- variables for Tokens and Colors
+- class names and some usage examples for Typography, Utils, Elements, and Components
+- URLs for brand assets
+
+The variables listed in Tokens and Colors are CSS variables which do work as expected in SCSS and SASS. As an example, for the variable `--s-color-cyan-400`, the following syntax needs to be used:
+
+```css
+.selector {
+  color: var(--s-color-cyan-400);
+}
+```
```

Comment:
For now, should we focus just on using Savile through its classes? I thought we'd decided to recommend that users only use the classes that Savile defines, not the variables (since these are mostly for internal use).

For assets - it might warrant a sub-section "Static Assets" where we could just say something like 

> To use Savile's static assets like images and icons, check out the `/assets` page.

Since importing Savile's CSS doesn't really "grant" access to the assets, they could be used even if Savile's CSS isn't used.

Reply:
Yes, I wavered on what to include/not to include here RE the variables. The reason I added the tid-bit about variables is, as a user I will always click on the first item in nav bar first, then second, etc. Well our first two are Tokens and Colors. And if you go to that page, you may wonder "what are these things/what can I do with them?". 

Maybe I should add a note at the top of each page - so for Tokens and Colors - something to the effect of "these are mainly used internally, but just incase you were wondering, we provide them here?" Then for the ones with classes, it seems a little more clear since we specify "class name" in the table header". Or maybe we should move Tokens and Colors to bottom of nav bar? Or maybe it's just me that would be confused by that🤣

And thank you for the note on assets, yes!

Reply:
I like the note "these are mostly used internally" for tokens & colors.